src/core/caniuse.js

```diff
@@ -103,60 +108,69 @@ function normalizeConf(conf) {
  * promises to get content for canIUse table
  * @param {Object} conf              normalized respecConfig.caniuse
  */
-async function canIUse(conf, cache) {
-  return new Promise(async (resolve) => {
+async function fetchAndCacheJson(conf) {
+  const cache = new IDBCache("respec-caniuse", ["caniuse"]);
```

Comment:
I'm still wondering if we should just add this up on line 23. We can spin up the database, even if we don't end up using it... it's cheap. WDYT? It's ok if we don't.

Reply:
It won't hurt. I'll do it.
We are calling `cache.ready` when needed anyway